### PR TITLE
Field wells

### PR DESCRIPTION
**Pull request by natoverse**
Adds a getFieldWells method to AppProfiles, which provide a spec for rendering custom wells in the tree to accept linked resource inputs. Right now the expectation is that this wells are used to link input tables and such.

### CODE REVIEW

**Comment by natoverse** [comment]
https://user-images.githubusercontent.com/720083/224842807-de910b48-ceca-4626-9403-015a1b7f8b98.mov

File: javascript/app-framework/tsconfig.json [hunk 1]
```diff
@@ -14,5 +14,5 @@
 		"esModuleInterop": false,
 		"isolatedModules": true
 	},
-	"include": ["src"]
+	"include": ["src", "../react/src/components/FieldWell/FieldWell.tsx"]
```
Comment (darthtrevino) [inline javascript/app-framework/tsconfig.json:17]: This doesn't look intentional

**Review by darthtrevino — APPROVED** [review]

**Comment by darthtrevino** [comment]
I'm not sure how I feel about the dropdown stylistically, but it's functional, and I'm okay with shipping it

File: javascript/app-framework/tsconfig.json (hunk 1 above)
Comment (natoverse) [reply javascript/app-framework/tsconfig.json:17]: No...no it does not

**Comment by natoverse** [comment]
> I'm not sure how I feel about the dropdown stylistically, but it's functional, and I'm okay with shipping it.
> 
> Maybe we can lighten up the dark border around the dropdown and remove or lighten up the chevron?

Fixed. It seemed like maybe a focus issue. I removed the border from the dropdown explicitly, and now it looks fine and has correct focus behavior.

**Pull request merged**